environments/docker.toml

```diff
@@ -239,6 +240,14 @@
         repository = "github.com/makerdao/vdb-mcd-transformers"
         migrations = "db/migrations"
         rank = "0"
+    [exporter.auction_file]
+        path = "transformers/events/auction_file/initializer"
+        type = "eth_event"
+        repository = "github.com/makerdao/vdb-mcd-transformers"
+        migrations = "db/migrations"
+        contracts = ["MCD_FLAP", "MCD_FLIP_BAT_A", "MCD_FLIP_ETH_A", "MCD_FLIP_SAI", "MCD_FLIP_TUSD_A",
```

Comment:
🎗️ just a note to remember to add new collateral types in the rebase.

Reply:
good call!